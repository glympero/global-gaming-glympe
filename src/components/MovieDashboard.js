import React from 'react';
import Title from './Title';
import MovieList from './MovieList';
import MovieListFilters from './MovieListFilters';
import Card from './Card';

const MovieDashboard = (props) => {

    return (
      <div>
        <Title title="Welcome to our Movies database" />
        <Card />
    </div>
    );
  };

export default MovieDashboard;