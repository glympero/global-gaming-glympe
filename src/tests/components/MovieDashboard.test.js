import React from 'react';
import { shallow } from 'enzyme';
import MovieDashboard from '../../components/MovieDashboard';

test('should render UserDashboard', () => {
    const wrapper = shallow(< MovieDashboard />);
    expect(wrapper).toMatchSnapshot();
});