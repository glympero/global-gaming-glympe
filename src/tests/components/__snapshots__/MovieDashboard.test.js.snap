// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render UserDashboard 1`] = `
<div>
  <Title
    title="Welcome to our Movies database"
  />
  <Clock />
</div>
`;
